### ISSUE TITLE
some questiones about  training  detail in  x3 and x4

### ISSUE THREAD

**Issue by jingtu180** [Zheng222/IDN-Caffe#15]
I can't achieve the effect of the paper in x3 and x4.can you share some details or parameters need to noticed about generation of training set and Test set in x3 and x4? your solver about x3 and x4 model?
thank you.

**Comment by z384141819** [comment]
I can't achieve the effect of the paper in x3 and x4.can you share some details or parameters need to noticed about generation of training set and Test set? your solver about x3 and x4 model?
thank you.

**Comment by Zheng222** [comment]
@z384141819 Hello, you should train the x3 (or x4) model by initializing the network using the x2 model parameters.

**Comment by z384141819** [comment]
train the x3 (or x4) model by initializing the network using the x2 model parameters.How do you set the learning rate and loss function？
first,I use 0.0001 and L1 in training size data, then use 0.00001 and L2 in fine-tuning size data. I can not reach your effect, especially in x4，The result of the experiment is very poor.




------------------ 原始邮件 ------------------
发件人: "Zheng Hui (惠政)"<notifications@github.com>; 
发送时间: 2019年4月26日(星期五) 上午8:24
收件人: "Zheng222/IDN-Caffe"<IDN-Caffe@noreply.github.com>; 
抄送: "张凯"<384141819@qq.com>; "Mention"<mention@noreply.github.com>; 
主题: Re: [Zheng222/IDN-Caffe] some questiones about  training  detail in x3 and x4 (#15)




@z384141819 Hello, you should train the x3 (or x4) model by initializing the network using the x2 model parameters.
